Document ability to comment out sections of release notes. (#137)
demisto/content-docs: docs/integrations/changelog.md

  - Adding/updating arguments
  - Updating outputs
  - Fixes for customer bugs
  
  
## Excluding Items

> Release notes are required to contain all items which have been changed documented within them. As such, validation will fail if detected items are removed from the generated release notes file.

Author: ```suggestion
Release notes are required to contain all items which have been changed included in the generated file. As such, validation will fail if detected items are removed from the generated release notes file.
```
